controllers/helpers.go

```diff
@@ -1120,3 +1121,29 @@ func GetClusterScoper(ctx context.Context, logger logr.Logger, c client.Client,
 
 	return nil, errors.Errorf("unsupported infrastructure type %q, should be AzureCluster or AzureManagedCluster", cluster.Spec.InfrastructureRef.Kind)
 }
+
+// AddBlockMoveAnnotation adds CAPI's block-move annotation returns whether or not the change had an effect.
```

Comment:
```suggestion
// AddBlockMoveAnnotation adds CAPI's block-move annotation and returns whether or not the annotation was added.
```
nit for more clarity